### PR TITLE
feat: transfer org page and other changes

### PR DESCRIPTION
**Pull request by Andrea-Villalobos**
**PR description + concerns**

Screenshots: 
image

image

1. Added the transfer org page.
Note: I temporarily used the login logs for checking the attempts in the confirm password. Asking for permission to add a table of password logs.

Scenario 1: Successful transfer of organization
https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/162780943/5e68da79-2051-48b4-8747-520120bf66b9

Scenario 2: Maximum attempts reached

https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/162780943/785479aa-e3ad-426d-8917-4f9e06e1f733

2. Vote Status can be set to abstained when the voter select 'abstain' in all positions.

3. The image placeholder of the abstained in the candidate preview is blank. Asking for new image placeholder. Thank you.

4. Added minor changes of pages.

### CODE REVIEW

**Comment by Carl-Tabuso** [comment]
> Asking for permission to add a table of password logs.

What is the purpose of password logs table?

**Comment by Andrea-Villalobos** [comment]
> > Asking for permission to add a table of password logs.
> 
> What is the purpose of password logs table?

I am currently using the login logs for counting the attempts for the confirm password before processing the transfer to another org. Once I had 5 attempts in the confirm password, then logged out, and try to login, I cannot try to login back so I was thinking it should have a separate logs table. I am not sure though but if you think it is not needed anymore, then I'll be just using the login logs. Thank you.

**Comment by Carl-Tabuso** [comment]
> I am currently using the login logs for counting the attempts for the confirm password before processing the transfer to another org. Once I had 5 attempts in the confirm password, then logged out, and try to login, I cannot try to login back so I was thinking it should have a separate logs table. I am not sure though but if you think it is not needed anymore, then I'll be just using the login logs. Thank you.

I do get it now. I had this issue before and what I did is once the user has reached the maximum of 5 incorrect attempts, we should show him a modal saying that he has reached the maximum limit and for security purposes, we will redirect him back to the landing page — we sort of think of this as a form of brute force attack or password guessing, hence why we destroy the current session, redirecting him to the landing page. You can check the implementation i made in my pr #172. I've also included there a new public method for `IpAddress` class called `countIpAddressAttempts`.

File: src/includes/verify-password.php
Comment (Carl-Tabuso) [inline src/includes/verify-password.php:1]: If you're creating a new class, ensure that it's inside the `classes` folder and does extend to the class you're using a method of, in this case you should extend your class to IpAddress.
```php
class PasswordVerify extends IpAddress {
  // Code here
}
``` 

I just realized that we have the same file name of `verify-password.php` inside the `includes` folder` on my latest pull request. OMFG

**Comment by Andrea-Villalobos** [comment]
Testing:


https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/162780943/3196ee99-91a4-4d52-8ac4-197000f122d6

**Comment by Carl-Tabuso** [comment]
Will provide my review in a bit.

**Review by Carl-Tabuso — APPROVED** [review]
Went over the code. Have no major issues found beside the `change-org-class.php`. Just ensure that you handle exception events such as when an organization is not picked. 
```php
    public function processForm($postData, $fileData) {
        if ($row) {
            // Check if organization is selected
            } else {
                // Organization is not selected, display an error message or handle it as needed
                //echo "Please select an organization.";
            }
        } else {
           // echo "Error: Voter ID not found.";
        }
    }
``` 

...and other form of exceptions in other methods of `FormHandler` class.

**Pull request merged**